### PR TITLE
C# Assignment 2 Completed

### PR DESCRIPTION
**Pull request by emtec-damodhar-jadhao**

### CODE REVIEW

File: CS_Assignment_2/Question_1.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+namespace CS_Assignment_2
+{
+    internal class Question_1
+    {
+        // Write a C# program to validate an Email ID using regex        
+        public bool CheckEmail(string email)
+        {
+
+            Regex emailregex = new Regex(@"^[a-zA-Z0-9+_.-]+@[a-zA-Z0-9.-]+$", RegexOptions.IgnoreCase);
```
Comment (nitish-poojari) [inline CS_Assignment_2/Question_1.cs]: this should fail if it doesnt contains . after @. For example xyz@gmailcom

File: CS_Assignment_2/Question_3.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        {
+            return id + " " + Name + " " +
+                   Education;
+        }
+
+        public void showStudent()
+        {
+            List<Question_3> students = new List<Question_3>
+            {
+                new Question_3{id=1,Name="Abhishek",Education="CSE" },
```
Comment (nitish-poojari) [inline CS_Assignment_2/Question_3.cs]: you should not create object of own class.

File: CS_Assignment_2/Question_5.cs [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        int emp_id;
+        string Emp_Name;
+        int Emp_Salary;
+        string Emp_Department;
+
+        public void ShowEmpData()
+        {
+            List<Question_5> employee = new List<Question_5>()
+    {
+        new Question_5{emp_id = 101, Emp_Name = "Amit",
```
Comment (nitish-poojari) [inline CS_Assignment_2/Question_5.cs]: dont create object of same class. Create another class

File: CS_Assignment_2/Question_1.cs (hunk 1 above)
Comment (emtec-damodhar-jadhao) [reply CS_Assignment_2/Question_1.cs]: updated regex

File: CS_Assignment_2/Question_3.cs (hunk 2 above)
Comment (emtec-damodhar-jadhao) [reply CS_Assignment_2/Question_3.cs]: updated changes

File: CS_Assignment_2/Question_5.cs (hunk 3 above)
Comment (emtec-damodhar-jadhao) [reply CS_Assignment_2/Question_5.cs]: Another class created

**Review by nitish-poojari — APPROVED** [review]

**Pull request merged**